3001-4000/LIT3034RepCh314.xml

```diff
@@ -66,18 +67,29 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
                   <ptr target="bm:Chaine1913Rep"/>
                   <citedRange unit="item">314</citedRange>
                </bibl>
+               <bibl>
+                  <ptr target="bm:Grebaut1935LesanaSab"/>
+               </bibl>
+            </listBibl>
+
+            <listBibl type="secondary">
+               <bibl>
+                  <ptr target="bm:HabtemichaelNosnitsin2007Lessanasab"/>
+               </bibl>
             </listBibl>
          </div>
          <div type="edition">
+           <note>The incipit follows <ref type="mss" corresp="EMIP00185"/>.</note>
+
             <div type="textpart" subtype="incipit" xml:lang="gez">
-              <ab>ጸሎት፡ በእንተ፡ ልሳነ፡ ሰብእ።</ab>
+              <ab>ጸሎት፡ በእንተ፡ ልሳነ፡ ሰብእ፡ ልሳነ፡ ምቀኛ፡ ወተንኮለኛ፡ ልሳነ፡ እስላም፡ ወክርስቲያን።</ab>
               <ab>
                 <lg>
```

Comment:
`<lg>` has not yet been used in BM, in such cases it would be good to open an issue. @PietroLiuzzo

Reply:
I can remove it for now, then. I guess it hasn't been used so far for verse text (መልክእ፡, ዓርኬ፡, ሰላም፡, etc.)? It seems to be what it's meant for (stanzas, refrains, etc.).

Reply:
I see no trouble with using it, it is just our project practice to discuss the introduction of elements not yet in use, to avoid uncoordinated improvisation.